docs/releases/v1.33.0.md

```diff
@@ -0,0 +1,267 @@
+
+    ```yaml
+    spec:
+      kubernetes:
+        masters:
+          kernelParameters:
+          - name: "fs.file-max"
+            value: "9223372036854775804"
+    ```
+
+- [[#425](https://github.com/sighupio/distribution/pull/425)] Adds trusted CA certificate support in OIDC authentication with self-signed certificates:
+  - `oidcTrustedCA` key under `spec.distribution.modules.auth` allows automatic provisioning of custom CA certificates for auth components.
+  - Adds secret generation and volume mounting for Gangplank, Pomerium, and Dex deployments.
+  - Supports `{file://path}` notation.
+
+    ```yaml
+    spec:
+      distribution:
+        modules:
+          auth:
+            oidcTrustedCA: "{file://my-ca.crt}"
+    ```
+
+- [[#428](https://github.com/sighupio/distribution/issues/428)] Configuration for Logging Operator's Fluentd and Fluentbit resources:
+  - Added new configuration options to the logging module that allows to set Fluentd's resources and replicas number and Fluentbit's resources. Example:
+  
+    ```yaml
+    spec:
+      distribution:
+        modules:
+          logging:
+            operator:
+              fluentd:
+                replicas: 1
+                  resources:
+                    limits:
+                      cpu: "2500m"
+              fluentbit:
+                resources:
+                  requests:
+                    memory: "1Mi"
+    ```
+
+- [[#429](https://github.com/sighupio/distribution/issues/429)] Control Plane taints for OnPremises clusters:
+  - Added new configuration option to set the control plane nodes taints at cluster creation time. Example:
+  
+    ```yaml
+    # custom taint. NOTE: the default taint won't be added, just the ones defined.
+    spec:
+      kubernetes:
+        masters:
+          taints:
+            - effect: NoExecute
+              key: soft-cell
+              value: tainted-love
+    ```
+  
+    ```yaml
+    # no taints
+    spec:
+      kubernetes:
+        masters:
+          taints: []
+    ```
+
+- [[#435](https://github.com/sighupio/distribution/pull/435)] Repository management lifecycle configuration for OnPremises provider:
+  - Added new boolean configuration fields for environments where package repositories are configured outside of furyctl.
+    - `spec.kubernetes.loadBalancers.manageRepositories`: Controls HAProxy repository setup
+    - `spec.kubernetes.advanced.containerd.manageRepositories`: Controls NVIDIA container toolkit's repository setup
+    - `spec.kubernetes.advanced.manageRepositories`: Controls Kubernetes package repository setup
+  - All fields are optional. If omitted, the system defaults to automatic repository management.
+
+    ```yaml
+    spec:
+      kubernetes:
+        loadBalancers:
+          enabled: true
+          manageRepositories: false
+        advanced:
+          manageRepositories: false
+          containerd:
+            manageRepositories: false
+    ```
```

Comment:
IMO is not clear to the reader from the text if this is the default or an example on how to disable it.

Seeing it now, I feel we should have named it `selfmanagedRepositories` because `manageRepositories` can be interpreted as "let furyctl manage them" or "I want to manage them"; but too late for that.